changelog.markdown

```diff
@@ -6,7 +9,7 @@
 * Updated to NSubsitute 1.8.1.0
 
 ## 2.6.0
-* Fixed counting the number of calls in the NSubstitute adapter when the call includes arguments (thanks to [Robert Anderson](https://github.com/shamp00))
+* Fixed counting the number of calls in the NSubstitute adapter when the call includes arguments (thanks to [Robert Anderson](https://github.com/shamp00))
```

Comment:
This is a line ending difference. The whole file is LF, but this line was CRLF. For the life of me I couldn't get any text editor to preserve the CRLF for me so it has been changed to an LF here.